application/controllers/ContactController.php

```diff
@@ -50,11 +50,9 @@ public function indexAction()
                 $this->redirectNow('__CLOSE__');
             })->on(ContactForm::ON_REMOVE, function (ContactForm $form) {
                 $form->removeContact();
-                /** @var FieldsetElement $contactElement */
-                $contactElement = $form->getElement('contact');
                 Notification::success(sprintf(
                     t('Deleted contact "%s" successfully'),
-                    $contactElement->getValue()
+                    $form->getValue('contact[full_name]')
```

Comment:
This cannot work.

Reply:
`$form->getValue('contact')['full_name']` should be used instead.

And please also add the change for `ON_SUCCESS`.